docs/source/README.md

```diff
@@ -7,30 +7,23 @@ This [document](../spec/sources.md) contains the source specification.
 ## Implementations
 
 There are various ways to implement Event Sources, as described in the
-[documentation](https://knative.dev/docs/eventing/samples/writing-receive-adapter-source/).
+[documentation](https://knative.dev/docs/eventing/samples/writing-event-source/).
 
-### Receive Adapters
+### Receive adapters
```

Comment:
I'm not sure I understand why we're no longer capitalizing this? AIUI the capitalization would match with the style typically implemented

@abrennan89  what's the correct approach in knative docs in general?

Reply:
Knative doc follows this style: https://developers.google.com/style/

Headers seem to be non-capitalized.

Reply:
This https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings ?
I think Receive Adapter is name of object should be capitalized everywhere used? Example from k8s docs for HPA: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/

Reply:
I'm sure the k8s doc is not perfect. If you don't mind I'd like to keep it the way it is now (lower case).

Reply:
I've given up blocking PRs over capitalization / formatting until we have a project consensus :shrug: :smile: 

I'm honestly not sure what is the best approach to take. There will probably be other occasions like this where one rule seems to contradict another, so what I think we really need is a sort of style guide / glossary that calls out specific terms and agreement on how we apply capitalization, formatting etc. across Knative. Let's follow up on Slack or with the WG leads / TOC maybe?

Everyone's comments on this thread are valid, so it just goes to illustrate the importance of coming up with a set of rules as a group IMHO.

To me, the following rules would ideally apply: capital letter for only the first word of a title, with the exception of objects. For example, I'd make this title "Receive Adapters", but, for example, if this title wasn't just that object, I'd capitalize it like "This title is about Receive Adapters".

Reply:
thanks for the input @abrennan89!
@lionelvillard could we just match the other titled `Receive Adapter` then and move on?

Reply:
ah sorry forgot about this one! Thanks!